GNUmakefile

```diff
@@ -132,7 +132,6 @@ gen:
 	rm -f internal/sweep/sweep_test.go
 	rm -f names/caps.md
 	rm -f names/*_gen.go
-	rm -f website/allowed-subcategories.txt
```

Comment:
?

Reply:
It's unneeded, since the generator truncates the file

Reply:
Most of these probably are

Reply:
👍